app/models/ext_management_system.rb

```diff
@@ -274,6 +274,17 @@ def connection_configurations=(options)
     options.each do |option|
       add_connection_configuration_by_role(option)
     end
+
+    drop_unused_connections(options)
+  end
+
+  def drop_unused_connections(options)
```

Comment:
See [note about renaming this](https://github.com/ManageIQ/manageiq/pull/8313/files#r62088243).